aws/internal/keyvaluetags/servicecatalog_tags.go

```diff
@@ -60,3 +60,38 @@ func ServiceCatalogProductUpdateTags(conn *servicecatalog.ServiceCatalog, identi
 
 	return nil
 }
+
+func ServicecatalogRecordKeyValueTags(tags []*servicecatalog.RecordTag) KeyValueTags {
+	m := make(map[string]*string, len(tags))
+
+	for _, tag := range tags {
+		m[aws.StringValue(tag.Key)] = tag.Value
+	}
+
+	return New(m)
+}
+
+func ServiceCatalogProvisionedProductUpdateTags(conn *servicecatalog.ServiceCatalog, identifier string, oldTagsMap interface{}, newTagsMap interface{}) error {
```

Comment:
Match capitalization.

```suggestion
func ServicecatalogProvisionedProductUpdateTags(conn *servicecatalog.ServiceCatalog, identifier string, oldTagsMap interface{}, newTagsMap interface{}) error {
```

Reply:
Hmmm... Also that's not used anywhere. :smh:

Reply:
removed